astropy/io/ascii/html.py

```diff
@@ -83,7 +83,14 @@ def process_lines(self, lines):
                                         'installed to read HTML tables')
 
         if 'parser' not in self.html:
-            soup = BeautifulSoup('\n'.join(lines))
+            try:
+                import warnings
+            except ImportError:
```

Comment:
Is this really necessary? `warnings` come from standard Python library, so why would it not be there?